lib/active_admin_sidebar/version.rb

```diff
@@ -1,3 +1,3 @@
 module ActiveAdminSidebar
-  VERSION = "0.1.0.rc2"
+  VERSION = '0.1.0.rc6'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.